CMakeLists.txt

```diff
@@ -85,7 +85,9 @@ if(LibOSRM_FOUND)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${LibOSRM_CXXFLAGS}")
 
   file(GLOB OSRM_EXECS ${LibOSRM_EXEC_DIR}/osrm-*)
-  file(COPY ${OSRM_EXECS} DESTINATION ${CMAKE_CURRENT_SOURCE_DIR}/src/bin)
+  install(FILES ${OSRM_EXECS} TYPE BIN 
+          PERMISSIONS OWNER_WRITE OWNER_READ GROUP_READ WORLD_READ 
+          OWNER_EXECUTE GROUP_EXECUTE WORLD_EXECUTE)
```

Comment:
so this worked now with permissions?

Reply:
Yep, this change allowed the executables to have execute permissions!